packages/alea-frontend/components/nap/GradingInterface.tsx

```diff
@@ -413,6 +431,26 @@ function GradingItemDisplay({
     refreshGradingInfo();
   }, [homeworkId, questionId, studentId]);
 
+  useEffect(() => {
+    const fetchProblem = async () => {
+      if (!questionMap[questionId]) {
+        try {
+          const uriWithoutId = questionId.replace(/en\?.*$/, 'en');
```

Comment:
It is not necessary that the questions will end with `en`. They could end with `de`, `fr`, `cn` etc. Check for last question mark instead.